File: Java/Program/Mahasiswa.java
package Latihan.Latihan1.Java.Program;

public class Mahasiswa {
    private String nama;
    private String nim;
    private String prodi;
    private String fakultas;

    public Mahasiswa(){
        this.nama = "";
        this.nim = "";
        this.prodi = "";
        this.fakultas = "";
    }

    public Mahasiswa(String nama, String nim, String prodi, String fakultas){
        this.nama = nama;
        this.nim = nim;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    // Get-set nama
    public String getNama(){
        return this.nama;
    }

    public void setNama(String nama){
        this.nama = nama;
    }
    
    // Get-set nim
    public String getNIM(){
        return this.nim;
    }

    public void setNIM(String nim){
        this.nim = nim;
    }
    // Get-set prodi
    public String getProdi(){
        return this.prodi;
    }

    public void setProdi(String prodi){
        this.prodi = prodi;
    }
    // Get-set fakultas
    public String getFakultas(){
        return this.fakultas;
    }

    public void setFakultas(String fakultas){
        this.fakultas = fakultas;
    }
}

File: CPP/Program/Mahasiswa.cpp
#include <iostream>
#include <string>

using namespace std;

class Mahasiswa{

    private:

    string nama;
    string nim;
    string prodi;
    string fakultas;

    public:
    
    // Constructor
    Mahasiswa(){
    }

    Mahasiswa(string nama, string nim, string prodi, string fakultas){
        this->nama = nama;
        this->nim = nim;
        this->prodi = prodi;
        this->fakultas = fakultas;
    }

    // Get-set Nama
    void setNama(string nama){
        this->nama = nama;
    }

    string getNama(){
        return this->nama;
    }

    // Get-set NIM
    void setNIM(string nim){
        this->nim = nim;
    }

    string getNIM(){
        return this->nim;
    }

    // Get-set Prodi
    void setProdi(string prodi){
        this->prodi = prodi;
    }

    string getProdi(){
        return this->prodi;
    }

    // Get-set Fakultas
    void setFakultas(string fakultas){
        this->fakultas = fakultas;
    }

    string getFakultas(){
        return this->fakultas;
    }

    bool comparison(string elemen, string remove){
        return elemen == remove;
    }

    // Deconstructor
    ~Mahasiswa(){
    }

};